### DISCUSSION TITLE
Adding Event Descriptions to Wiki

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2689] [Q&A] [answered]
I was looking to add short descriptions of events to the wiki but I wasn't sure what the best way to do it would be. Adding another column to the table makes it look a bit bloated and hard to read in my opinion. I was wondering if anyone had any suggestions on how this could be accomplished better. [You can see what I'm talking about in this example here](/Denneisk/wire/wiki/Expression-2-Events-(demo)#-list-of-events).

I was thinking of mimicking the regular E2 documentation with headings for each event, maybe even in a separate page so the regular event page can stay concise and expository.

Also, event E2helper when?

**Comment by vurvdev** [comment, marked as answer]
I think you could go ahead with having two columns, and remove the `Replacing` column.

After all events aren't really new anymore, so there's more new people learning it than people migrating from triggers as of now.

Just include that it replaced X functions in the description. Then the description would have a lot more space and look more natural.
